### DISCUSSION TITLE
Repo

### DISCUSSION THREAD

**Discussion by ikagwu2020** [community/community#81600] [Repositories]
### Select Topic Area

Show & Tell

### Body

Hi, please i cant push my code from vscode to github.
this is the error message: remote: Permission to ikagwu2020/web-consulting.git denied to ikagwu

**Comment by anuragjain-git** [comment]
1. Check GitHub Authentication:
Make sure you are authenticated with the correct GitHub account in Visual Studio Code. You can check and update this by going to the bottom right of the window where it says "Signed in as" in the status bar. If it's not the correct account, you may need to sign out and sign in with the correct one.
2. Verify Repository URL:
Double-check that the remote repository URL is correct. Open a terminal in Visual Studio Code and run the following command:

-Check the current remote repository URL
git remote -v

-If needed, update the remote repository URL
git remote set-url origin <paste_repository_url_here>
